### ISSUE TITLE
The Property 'ShowCategory' cannot be found on this object

### ISSUE THREAD

**Issue by winberrym** [dfinke/ImportExcel#258]
Repro:

```
Import-Module ImportExcel
$data = get-service
$file = "C:\Temp\Stuff\TestExport.xlsx"
$base = @{
    SourceWorksheet = 'gsv'
    PivotData = @{'Status'='count'}
    IncludePivotChart = $true
}

$ptd = [ordered]@{}
$ptd += New-PivotTableDefinition @base servicetype -PivotRows servicetype -ChartType Area3D
$ptd += New-PivotTableDefinition @base status -PivotRows status -ChartType PieExploded3D
$ptd += New-PivotTableDefinition @base starttype -PivotRows starttype -ChartType BarClustered3D
$ptd += New-PivotTableDefinition @base canstop -PivotRows canstop -ChartType ConeColStacked

$data | Export-Excel -Path $file -WorkSheetname gsv -PivotTableDefinition $ptd
```
Output:

```
Failed exporting worksheet 'gsv' to 'C:\Temp\Stuff\TestExport.xlsx': The property 'ShowCategory' cannot be found on this object. 
Verify that the property exists and can be set.
At C:\Program Files\WindowsPowerShell\Modules\ImportExcel\4.0.8\Export-Excel.ps1:979 char:13
+             throw "Failed exporting worksheet '$WorkSheetname' to '$P ...
+             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : OperationStopped: (Failed exportin...and can be set.:String) [], RuntimeException
    + FullyQualifiedErrorId : Failed exporting worksheet 'gsv' to 'C:\Temp\Stuff\TestExport.xlsx': The property 'ShowCategory' cannot be found on this object. Verify that the property exists and can be set.
 
```

**Comment by dfinke** [comment]
```powershell
if($chart.DataLabel) {
    $chart.DataLabel.ShowCategory = [boolean]$item.value.ShowCategory
    $chart.DataLabel.ShowPercent  = [boolean]$item.value.ShowPercent
}
```

**Comment by winberrym** [comment]
Thanks!

**Comment by dfinke** [comment]
fixed dc4b66fffee9f1d13b22f1b82fd1dc562893bed9
